### ISSUE TITLE
Install of taglib fails on Raspbian/Jessie

### ISSUE THREAD

**Issue by vrtisworks** [savonet/ocaml-taglib#2]
Trying to do an install of taglib on Rasbian/Jessie, and I get the following errors.  Note that the message says to use opam depexts to check, but when I ran that, nothing was listed (run of depexts is at the bottom of the console listing below)

`
liquidsoap@bigpi:~ $ opam install taglib
The following actions will be performed:
  ∗  install taglib 0.3.2

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[default] https://opam.ocaml.org/archives/taglib.0.3.2+opam.tar.gz downloaded

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[ERROR] The compilation of taglib failed at "./configure --prefix
        /home/liquidsoap/.opam/system".
Processing  1/1: [taglib: ocamlfind remove]
#=== ERROR while installing taglib.0.3.2 ======================================#
# opam-version 1.2.2
# os           linux
# command      ./configure --prefix /home/liquidsoap/.opam/system
# path         /home/liquidsoap/.opam/system/build/taglib.0.3.2
# compiler     system (4.01.0)
# exit-code    1
# env-file     /home/liquidsoap/.opam/system/build/taglib.0.3.2/taglib-23290-d2d111.env
# stdout-file  /home/liquidsoap/.opam/system/build/taglib.0.3.2/taglib-23290-d2d111.out
# stderr-file  /home/liquidsoap/.opam/system/build/taglib.0.3.2/taglib-23290-d2d111.err
### stdout ###
# [...]
# checking for ocamlfind... ocamlfind
# checking for ocaml standard library path... /usr/lib/ocaml
# checking for caml/threads.h... yes
# checking for gcc option to produce PIC... -fPIC
# checking for g++... g++
# checking whether we are using the GNU C++ compiler... yes
# checking whether g++ accepts -g... yes
# checking for pkg-config... /usr/bin/pkg-config
# checking pkg-config is at least version 0.9.0... yes
# checking whether pkg-config knows about taglib >= 1.6... 
### stderr ###
# configure: error: taglib.pc not found.. Do you need to set PKG_CONFIG_PATH?

=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
The following actions failed
  ∗  install taglib 0.3.2
No changes have been performed

=-=- taglib.0.3.2 troobleshooting -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
=> This package relies on external (system) dependencies that may be missing.
   `opam depext taglib.0.3.2' may help you find the correct installation for
   your system.
liquidsoap@bigpi:~ $ opam depext taglib.0.3.2
# Detecting depexts using flags: arm armv7 linux raspbian
# No extra OS packages requirements found.
liquidsoap@bigpi:~ $ ^C

`

**Comment by vrtisworks** [comment]
Note:  I did an opam show taglib and determined that it was needed libtaglib, so I installed libtaglib-ocaml-dev via apt-get, and then tried to install taglib again.  This time it worked.

Also, I tried this with 1.2.0 of opam and had a similar problem (though with slightly different wording on some of the errors).  So I downloaded and installed 1.2.2 of opam and tried that.  That produced the console log shown above.
